### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kludex**
I think the `scripts` folder was removed at some point?

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ During development, if we make edits to the content we'd like the browser to aut
 
 - Clone the [`arel`](https://github.com/florimondmanca/arel) repository.
 - Make sure you are in the repository root directory.
-- Install dependencies: `$ scripts/install`.
+- Install dependencies: `$ pip install -r requirements.txt`.
```
Comment (florimondmanca) [inline example/README.md:13]: Yup, it became `make install` :)

File: example/README.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ During development, if we make edits to the content we'd like the browser to aut
...
 
 ## Usage
 
-- Start the server: `$ scripts/serve`.
+- Start the server: `$ uvicorn server.app:app`.
```
Comment (florimondmanca) [inline example/README.md:17]:
```suggestion
- Start the server: `$ make serve`.
```

File: example/README.md (hunk 1 above)
Comment (florimondmanca) [inline example/README.md:13]:
```suggestion
- Install dependencies: `$ make install`.
```

**Review by florimondmanca — COMMENTED** [review]
Yup, it became `make install` :)

**Review by florimondmanca — APPROVED** [review]

**Comment by florimondmanca** [comment]
Seems like a change in mypy breaks CI. We can look into that later, as it's an unrelated thing. Thanks @Kludex!

**Comment by Kludex** [comment]
Wow! I didn't even see a Makefile there! :sweat_smile:

**Comment by florimondmanca** [comment]
I've been slowly switching my "scripts" workflow to using `Makefile` over the past year. Turns out `make` is a pretty handy tool for basic commands!

**Pull request merged**